### ISSUE TITLE
Issue ANPR C019_ 404 not found

### ISSUE THREAD

**Issue by valeriobiondo** [italia/anpr#4609]
Buongiorno, siamo riusciti a generare il token ma stiamo riscontrando delle difficoltá nella chiamata dell'e-service, viene rilevato l'errore 404_not found.

Riporto il messaggio d'errore : 

<404 Not Found
<content-type:application/problem+json
<x-global-transaction-id:3f7e3e2d66991f756848556d
<x-backside-transport:FAIL FAIL
<connection:close
 
Stiamo utilizzando il seguente URL usando la modalitá "POST"  con l'autenticazione del bearer token

https://modipa-val.anpr.interno.it/govway/rest/in/MinInternoPortaANPR-PDND/C019-servizioAccertamentoEsistenzaVita/v1/anpr-service-e002

Trascriviamo anche la "response" 
"type": "
[https://govway.org/handling-errors/404/NotFound.html"](https://govway.org/handling-errors/404/NotFound.html%22)
,
  "title": "NotFound",
  "status": 404,
  "detail": "Unknown API Request",
  "X-Global-Transaction-ID": "3f7e3e2d66aa47ae78c9d97d"

Potreste gentilmente fornire dei chiarimenti riguardanti l'header? Il token é sufficiente per convalidare l'accesso all'API/e-service? 

Distinti Saluti

**Comment by afrontera85** [comment]
Salve, i possibili motivi del 404 Unknown API Request sono:
Non vi state presentando con il voucher pdnd nella richiesta (nell’attributo authorization)
Il voucher che state presentando nella richiesta non è valido
Non state chiamando la url con il metodo http giusto (POST) - che non è il vostro caso 
Comunque nella issue #4488 c'è un client che consigliamo sempre di usare

**Comment by valeriobiondo** [comment]
Vi ringraziamo per il cortese riscontro. 
Possiamo gentilmente chiedervi se attraverso il client indicato nella issue #4488 é possibile effettuare richieste massive? ( da intendersi come unica richiesta attraverso cui effettuare verifiche riferiti a diversi codici fiscali) 
Distinti Saluti

**Comment by afrontera85** [comment]
il client fornito è di esempio per effettuare singole chiamate

**Comment by valeriobiondo** [comment]
Sarebbe disponibile un client per effettuare richieste riferite a piú codici fiscali? 
Distinti Saluti

**Comment by afrontera85** [comment]
No, il client fornito è meramente esemplificativo di come implementare i profili di sicurezza di cui necessitano gli eservice di ANPR. Ma comunque non è possibile al momento un uso massivo del servizio C019 , lo è invece per C030 (#4599) e C007 (verifica esistenza in vita, resituisce S o N). 
Per i dettagli la invito a consultare la documentazione pubblicata sulla PDND

**Comment by Alexei-Dev** [comment]
Hi,

We have this problem now when trying to access c019 from the java client,

we are getting this response:
GovWay-Transaction-ID: d6e854af-5a23-11ef-88d8-005056ae1884
{"type":"https://govway.org/handling-errors/400/InteroperabilityInvalidRequest.html","title":"InteroperabilityInvalidRequest","status":400,"detail":"Received request is not conform to the required interoperability profile","govway_id":"d6e854af-5a23-11ef-88d8-005056ae1884"}

using:
baseurlapi=https://modipa-val.anpr.interno.it/govway/rest/in/MinInternoPortaANPR-PDND/C019-servizioAccertamentoEsistenzaVita/v1/anpr-service-e002

audTokenAgidJwtSignature=https://modipa-val.anpr.interno.it/govway/rest/in/MinInternoPortaANPR/C019-servizioAccertamentoEsistenzaVita/v1


Please let us know what we could be doing incorrectly.
